### PR TITLE
Wp caro/improve crud category endpoint

### PR DESCRIPTION
**Pull request by Carolina-Vacaflor**
Hello. I have improved tests for category endpoint. Please review my code.

### CODE REVIEW

File: wordpress/tests/common_keywords/categories/categories.robot
Comment (celestepalet) [inline wordpress/tests/common_keywords/categories/categories.robot:1]: Please left a empty line at the end of the code. 
It is not neccesary aend the expected result in get requests
It is not neccesary send the sintax format_json when uses " get_format_response", the default format is json 
When uses "Create dictionary", please put the second word whit  uppercase

Comment (celestepalet) [inline wordpress/tests/common_keywords/categories/categories.robot:1]: Please left a empty line at the end of the code.
It is not neccesary aend the expected status in get requests
It is not neccesary send the sintax format_json when uses " get_format_response", the default format is json
When uses "Create dictionary", please put the second word whit uppercase

Comment (Carolina-Vacaflor) [reply wordpress/tests/common_keywords/categories/categories.robot:1]: done!

Comment (celestepalet) [reply wordpress/tests/common_keywords/categories/categories.robot:1]: Thaks!

**Pull request merged**